content/en/blog/_posts/2021-07-21-memory-manager-beta.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: blog
+title: "Kubernetes Memory manager moves to beta"
+date: 2021-07-21
+slug: kubernetes-1-22-feature-topoloy-manager-beta
```

Comment:
```suggestion
slug: kubernetes-1-22-feature-memory-manager-beta
```
Should the slug include MemoryManager instead of TopologyManager

Reply:
done